src/main/java/edu/rosehulman/quota/controller/DeleteUserController.java

```diff
@@ -0,0 +1,34 @@
+package edu.rosehulman.quota.controller;
+
+import edu.rosehulman.quota.Database;
+import edu.rosehulman.quota.client.SharedServiceClient;
+import edu.rosehulman.quota.model.User;
+import org.apache.http.HttpException;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+public class DeleteUserController implements Route {
+
+  @Override
+  public Object handle(Request request, Response response) throws Exception {
+    String partnerId = request.params(":partnerId");
+    String productId = request.params(":productId");
+    String userId = request.params(":userId");
+
+    User user = new User();
+    user.setPartnerId(partnerId);
+    user.setProductId(productId);
+    user.setUserId(userId);
+
+    // delete the given user
+    Database.getInstance().deleteUser(user);
```

Comment:
If the user does not exist we want to return a 404. `deleteUser` returns a `boolean`, so check the boolean and if it is false, return a 404.